ui/src/message_popup/author/edit_question_page.jsx

```diff
@@ -1,8 +1,8 @@
-/* @flow weak */
```

Comment:
Flow was having issues with the props possibly being undefined. I couldn't fix it with setting the props as required because then React was having issues because I passed them in as a cloned element using `React.cloneElement`. I also tried using to prevent the props from possibly being undefined `getDefaultProps` but flow wouldn't recognize that either. For now, I disabled flow on this file and two others with the same issue.